### PR TITLE
Added webinars page & update page content

### PR DESCRIPTION
**Pull request by zprood**
## Done

- Removing the date of webinar pages 
- Adding a new page for webinar collection 
- Updating home page content : case study/whitepaper, 20.04 laptop image, webinars page section

### CODE REVIEW

**Comment by SirSamTumless** [comment]
@carkod Could you review this please?

File: templates/webinars/index.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+
+
+
+</div>
+
+
+
+
+{% endblock %}
```
Comment (carkod) [inline templates/webinars/index.html:112]: @zprood Can you remove all the additional space and end the file with a new line please? Thank you.

**Review by carkod — APPROVED** [review]
Just one comment.

File: templates/webinars/index.html (hunk 1 above)
Comment (zprood) [reply templates/webinars/index.html]: @carkod Yes, done. Thank you.

**Pull request merged**